package-overrides/npm/dot-prop@4.0.0.json

```diff
@@ -0,0 +1,3 @@
+{
+  "format": "es6"
```

Comment:
Done.
Out out curiosity, what is the difference between `"format": "esm"` and `"format": "es6"` ?
I always use the latter: should I always use the former instead ?